src/lib/theme/plugin.ts

```diff
@@ -11,6 +11,7 @@ export const variables = {
   '--color-brand': rgb(getColor('indigo', 800)),
   '--color-interactive': rgb(getColor('indigo', 600)),
   '--color-interactive-hover': rgb(getColor('indigo', 700)),
+  '--color-secondary-hover': rgb(getColor('slate', 50)),
```

Comment:
Can we use `--color-surfacec-interactive-secondary` instead? It's currently slate 100 for light and slate 800 for dark, but we could change it to slate 50 and 900 if needed

Reply:
Good call! Thanks Ross!